docs/installing-northstar/using-northstar/playing-on-linux.md

```diff
@@ -135,16 +59,6 @@ While playing with LatencyFleX, **VSync and Adaptive Super Sampling must be disa
 
 > Read Lutris troubleshooting of [common issues with Origin](https://github.com/lutris/docs/blob/master/Origin.md) first.
 
-### Blank console
-
-This problem is caused due to missing fonts on your Titanfall 2 wine prefix, you will need [winetricks](https://github.com/Winetricks/winetricks) or [protontricks](https://github.com/Matoking/protontricks) depending on your installation. Follow these steps to install:
-
-1. Close all Titanfall/Origin processes.
-2. If you are using Lutris select your Titanfall 2 installation and click '▲' -> Winetricks. On Proton you can use `protontricks 1237970 --gui`
-3. 'Select the default wineprefix' -> 'Install a font' -> Check the packages `lucida` and `arial`.
-4. Click OK and wait for everything to install, if done correctly winetricks will popup again.
-5. You can now close it and launch the game.
```

Comment:
Maybe keep blank console header around for a bit longer but just change the content to say that the issue was caused by missing font and fixed on `v1.9.2` or higher of Northstar.

Reply:
You got it.

Reply:
Addressed in latest commit.